CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.0.1]
+
+### Fixed
+- A bug that would cause some INSAR jobs, with `apply-water-mask`, to fail with a `shapely.errors.TopologicalError`.
```

Comment:
```suggestion
- A bug that would cause some INSAR jobs, with `apply-water-mask` set to `True` to fail with a `shapely.errors.TopologicalError`.
```

Reply:
Want to be a bit more explicit about the conditions under which the error occurs.